challenges.yml

```diff
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
+              transition: height;
+
+              &.tall {
+                  height: 400px;
+              }
+          }
+          </style>
+          ```
+        c: |
+          ```vue
+          <style lang="scss" module>
+          .component {
+              height: 200px;
+              transition: height 1s;
+
+              &.tall {
+                  height: 400px;
+              }
+          }
+          </style>
+          ```
+        d: |
+          ```vue
+          <style lang="scss" module>
+          .component {
+              height: 200px;
+              transition: height 1s;
+          }
+          </style>
+          ```
+      correct: c
+
+extending_css_transitions_2:
+  question: |
+    When we transition an elements opacity and try to show or hide this
+    element using `v-show`, our transition breaks as shown in the video.
+    Why is this so?
+  explanation: |
+    Since `v-show` changes the `display` property from `none` to `inherit`,
+    our transition over the `opacity` property has no effect in the DOM!
+    Notice that transitioning over the `display` property does not provide
+    a fading effect as we'd expect. This is important when talking about
+    Vue transitions later.
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          Because Vue's `v-show` directive overrides any class that deals
+          with an element's opacity or display properties.
+        b: |
+          Because Vue's `v-show` directive cancels any transition rules
+          that we apply to the element.
+        c: |
+          Because Vue's `v-show` directive makes the `opacity` property
+          jump from `1` to `0` immediately.
+        d: |
+          Because Vue's `v-show` directive works on the `display` property
+          of an element, which is not the property we're transitioning.
+      correct: d
+
+extending_transitions:
+  question: |
+    Our component that shows or hides a disclaimer message upon buying
+    pizza slices has made a big impact in our customer's health! Yay!
+
+    But our CEO is not very happy with the visual results... Nay!
+
+    He now wants us to just simply fade in or out such message when
+    a user wants to buy too much pizza. We can use Vue transitions for
+    that!
+
+    Here's the initial component:
+
+    ```vue
+    <template>
+        <div>
+            <button @click="addPizzaToCart" />
+
+            <p v-show="showDisclaimer">
+                <strong>Disclaimer</strong>: Eating too much pizza can be
+                bad for your health!
+            </p>
+        </div>
+    </template>
+
+    <script>
+    import { buyPizza } from '@/services/buy-pizza';
+
+    export default {
+        name: 'PizzaCheckout',
+        data() {
+            return {
+                showDisclaimer: false,
+            };
+        },
+        methods: {
+            async addPizzaToCart() {
+                await buyPizza();
+            }
+        },
+    };
+    </script>
```

Comment:
I wonder if we could avoid showing the `script` part entirely? And just say something like:

> How could we make the message fade in and out whenever the `showDisclaimer` variable changes?